Cargo.lock

```diff
@@ -5541,7 +5541,7 @@ dependencies = [
  "solana-rpc",
  "solana-runtime",
  "solana-sdk",
- "tokio 1.10.1",
+ "tokio 1.11.0",
```

Comment:
Interesting. Did your changes pull in these Cargo modifications? Or maybe this was to fix a CI check?

Reply:
i suspect someone will update master. I didn't cause it that I can imagine.

Reply:
I merged bumps to futures and tokio recently. Perhaps those prs missed bumping replica-lib, for some reason 🤔 
Anyway, shouldn't be an issue to pull in here.